scripts/ci/push-image.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# push-image.sh pushes the docker image to the docker registry
+
+{(# run in a subshell
+
+set -e          # make any error fail the script
+set -u          # make unbound variables fail the script
+set -o pipefail # make any pipe error fail the script
+
+# RELEASE_TAG is the release tag that we want to release
+RELEASE_TAG="${1}"
+
+if [ -z "$RELEASE_TAG" ]; then
+  branch=$(git rev-parse --abbrev-ref HEAD)
+  commit=$(git rev-parse --short HEAD)
+  if [ "$branch" = "master" ]
+  then RELEASE_TAG="latest"
+  elif [ "$branch" = "main" ]
+  then RELEASE_TAG="main"
```

Comment:
Why different release tags for `master` and `main`? Would it make sense to use the same tag for both cases?